providers/os/connection/snapshot/partition.go

```diff
@@ -21,6 +21,17 @@ type PartitionInfo struct {
 	Uuid string
 	// (optional) MountPoint is the partition mount point
 	MountPoint string
+
+	// (optional) MountOptions are the mount options
+	MountOptions []string
+	// (optional) Bind adjusts the root for FS connection
+	Bind string
```

Comment:
If i understand correctly, this is for OS trees where the root is not at the mountpoint but somewhere else? Maybe we call this `RootDir` or something? Should we make this always filled in and for:
* os tree partitions its probably not equal to `Mountpoint`
* for other parts, its usually equal to `Mountpoint`

but we always use this property to initiate FS scans from, so that code is consistent and its just the `RootDir` logic that varies

Reply:
Thing is that MountPoint might be changing, but Bind shouldn't
Also, there are use cases where we aren't interested in root but whole fs
I propose we can add it as a method, so it's consistent